packages/validator-rules/README.md

```diff
@@ -60,7 +61,7 @@ The rules used closely follow the proto definitions from [validator.proto](https
 
 Specifically:
 
--   The `raw` property is unprocessed [ValidatorRules](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L643)
+-   The `raw` property is unprocessed [ValidatorRules](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L643), the same format used by `https://cdn.ampproject.org/v0/validator.json`
```

Comment:
nit: turn this into a link

Reply:
Don't think that makes sense, clicking that link is probably not very useful, it's more meaningful as an identifier.

Reply:
I'd want to click it :-). But I agree unformatted JSON is not helpful.